packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -677,7 +677,8 @@ function actOnColorScheme(colorScheme: string | null | undefined) {
   if (!colorScheme) {
     return;
   }
-  document.documentElement.style.colorScheme = colorScheme;
+  document.documentElement.style.colorScheme = colorScheme || "unset";
+  document.body.style.background = "transparent";
```

Comment:
I meant that we shouldn't modify anything in the iframe. Inside the iframe, cal.com code is there, we already handle that correctly.